### PR TITLE
improve hello-minikube page for dashboard

### LINKED ISSUE #28639: Improvement for k8s.io/docs/tutorials/hello-minikube/
> To stop the proxy, run Ctrl+C to exit the process. The dashboard remains running.

Nope, when you press Ctrl+C after running `minikube dashboard` the dashboard stops working.

### PR DESCRIPTION
**Pull request by chenxuc**
fix #28639

clarify dashboard command to avoid confusing beginners.

### CODE REVIEW

**Comment by chenxuc** [comment]
@jimangel @zparnold Could you please review this PR? Thanks a lot!

File: content/en/docs/tutorials/hello-minikube.md [hunk 1]
```diff
@@ -60,11 +60,16 @@ If you installed minikube locally, run `minikube start`. Before you run `minikub
...
-The `dashboard` command enables the dashboard add-on and opens the proxy in the default web browser. You can create Kubernetes resources on the dashboard such as Deployment and Service.
+The `dashboard` command enables the dashboard add-on and opens the proxy in the default web browser.
+You can create Kubernetes resources on the dashboard such as Deployment and Service.
 
 If you are running in an environment as root, see [Open Dashboard with URL](#open-dashboard-with-url).
 
-To stop the proxy, run `Ctrl+C` to exit the process. The dashboard remains running.
+By default, the dashboard is only accessible from within the internal Kubernetes virtual network.
+The `dashboard` command creates a temporary proxy to make the dashboard accessible from outside the Kubernetes virtual network.
+To stop the proxy, run `Ctrl+C` to exit the process.
```
Comment (celestehorgan) [inline content/en/docs/tutorials/hello-minikube.md:70]:
```suggestion
By default, the dashboard is only accessible from within the internal Kubernetes virtual network.
The `dashboard` command creates a temporary proxy to make the dashboard accessible from outside the Kubernetes virtual network.

To stop the proxy, run `Ctrl+C` to exit the process.
```

**Review by celestehorgan — COMMENTED** [review]
/lgtm

**Comment by chenxuc** [comment]
@celestehorgan Thanks for the review. I've updated the doc with your comment.

**Pull request merged**